Tests/test_imageqt.py

```diff
@@ -2,10 +2,13 @@
 
 import pytest
 
-from PIL import ImageQt
-
 from .helper import assert_image_similar, hopper
 
+with warnings.catch_warnings() as w:
+    warnings.simplefilter("always", category=DeprecationWarning)
```

Comment:
```suggestion
    warnings.simplefilter("ignore", category=DeprecationWarning)
```
Should this not be "ignore" like the other tests except for test_deprecated_imageqt.py?

Reply:
Yep